bpython/translations/bpython.pot

```diff
@@ -61,42 +61,14 @@ msgstr ""
 msgid "File to execute and additional arguments passed on to the executed script."
 msgstr ""
 
-#: bpython/cli.py:320 bpython/curtsiesfrontend/interaction.py:107
```

Comment:
The `#: 'bpython/curtsiesfrontend/interaction.py:107` should stay.

Reply:
Oops, bit too heavy on the delete.